src/routes/version_creation.rs

```diff
@@ -167,13 +167,28 @@ async fn version_create_inner(
                 file_name: uploaded_text.file_name.clone(),
             });
 
-            // TODO: do a real lookup for the channels
             let release_channel = match version_create_data.release_channel {
                 VersionType::Release => models::ChannelId(1),
                 VersionType::Beta => models::ChannelId(3),
                 VersionType::Alpha => models::ChannelId(5),
             };
```

Comment:
Still an issue.  This should probably be shared between version creation and mod creation so that they are consistent.